quinn-proto/src/connection.rs

```diff
@@ -2600,7 +2602,11 @@ where
     ///
     /// This does not ensure delivery of outstanding data. It is the application's responsibility to
     /// call this only when all important communications have been completed, e.g. by calling
-    /// `finish` on outstanding streams and waiting for the corresponding `StreamFinished` event.
+    /// [`Connection::finish`] on outstanding streams and waiting for the corresponding
```

Comment:
Let's also keep documentation-only improvements in another separate commit.

Reply:
One reason I changed the documentation is that I could not tell if, should `Connection::write` and/or `Connection::finish` return `WriteError::Stopped`, a `StreamFinished` event will be received.  I assume the answer is “no”, but this should be made clearer.

Reply:
I agree, and this is a good change. It's just easier to review logically independent changes for correctness if they're in separate commits, and easier to bisect changes in the future.